lib/ui/views/app_selector/app_selector_viewmodel.dart

```diff
@@ -69,6 +70,21 @@ class AppSelectorViewModel extends BaseViewModel {
     return true;
   }
 
+  Future<void> searchSuggestedVersionOnWeb(
+      {required String packageName,}) async {
+    final String suggestedVersion = getSuggestedVersion(packageName);
+    String stringUrl = 'https://www.google.com/search?q=$packageName'
+        '+apk';
```

Comment:
Do not hardcode Google. If possible "launch" a browser search using the default browser.

Reply:
A quick Google search yielded this: https://stackoverflow.com/questions/2201917/how-can-i-open-a-url-in-androids-web-browser-from-my-application

Reply:
I am also failing to find solutions to this, this sounds like a really stupid limitation of Android

Reply:
https://developer.android.com/guide/components/intents-common#SearchWeb

Reply:
@oSumAtrIX @Ushie This might require native platform implementation, seperate for iOS and Android.